sdk/cognitivelanguage/Azure.AI.Language.Text/api/Azure.AI.Language.Text.net8.0.cs

```diff
@@ -122,21 +122,10 @@ protected override void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter write
         string System.ClientModel.Primitives.IPersistableModel<Azure.AI.Language.Text.AllowOverlapEntityPolicyType>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.AI.Language.Text.AllowOverlapEntityPolicyType>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
-    public partial class AnalyzeTextDynamicClassificationResult : Azure.AI.Language.Text.AnalyzeTextResult, System.ClientModel.Primitives.IJsonModel<Azure.AI.Language.Text.AnalyzeTextDynamicClassificationResult>, System.ClientModel.Primitives.IPersistableModel<Azure.AI.Language.Text.AnalyzeTextDynamicClassificationResult>
-    {
-        internal AnalyzeTextDynamicClassificationResult() { }
-        public Azure.AI.Language.Text.DynamicClassificationResult Results { get { throw null; } }
-        protected override void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
-        Azure.AI.Language.Text.AnalyzeTextDynamicClassificationResult System.ClientModel.Primitives.IJsonModel<Azure.AI.Language.Text.AnalyzeTextDynamicClassificationResult>.Create(ref System.Text.Json.Utf8JsonReader reader, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
-        void System.ClientModel.Primitives.IJsonModel<Azure.AI.Language.Text.AnalyzeTextDynamicClassificationResult>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
-        Azure.AI.Language.Text.AnalyzeTextDynamicClassificationResult System.ClientModel.Primitives.IPersistableModel<Azure.AI.Language.Text.AnalyzeTextDynamicClassificationResult>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
-        string System.ClientModel.Primitives.IPersistableModel<Azure.AI.Language.Text.AnalyzeTextDynamicClassificationResult>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
-        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.AI.Language.Text.AnalyzeTextDynamicClassificationResult>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
-    }
     public partial class AnalyzeTextEntitiesResult : Azure.AI.Language.Text.AnalyzeTextResult, System.ClientModel.Primitives.IJsonModel<Azure.AI.Language.Text.AnalyzeTextEntitiesResult>, System.ClientModel.Primitives.IPersistableModel<Azure.AI.Language.Text.AnalyzeTextEntitiesResult>
     {
         internal AnalyzeTextEntitiesResult() { }
-        public Azure.AI.Language.Text.EntitiesResult Results { get { throw null; } }
+        public Azure.AI.Language.Text.EntitiesWithMetadataAutoResult Results { get { throw null; } }
```

Comment:
EntitiesWithMetadataAutoResult => this class name is very ambiguous, can we simplify